api-gen/rendering/templates/parameter.tsx

```diff
@@ -0,0 +1,20 @@
+/** @jsx h */
+import {h} from 'preact';
+import {ParameterEntryRenderable} from '../entities/renderables';
+import {RawHtml} from './raw-html';
+
+
+/** Component to render a function or method parameter reference doc fragment. */
+export function Parameter(props: {param: ParameterEntryRenderable}) {
```

Comment:
I know this will end up being a Preact construct, but do you think an explicit type can be added? like `h.JSX.Element`?

Reply:
I'd prefer to rely on the inferred type for now since it's not a "public" API; any type mismatches should be caught in the normal composition of components.

Reply:
Fair enough. Explicit types just speed up type-checking and IDE's so I'm generally a fan of those. But this is a nit